### PR TITLE
Testing framework does correct pass the parseoptions argument.

### PR DESCRIPTION
**Pull request by AdamSpeight2008**
Pass through the `parseOptions` to the compilation.

### CODE REVIEW

**Review by AdamSpeight2008 — COMMENTED** [review]
This may trigger an assert where `.ParseOption IsNot Nothing`.

File: src/Compilers/Test/Utilities/VisualBasic/BasicTestBase.vb [hunk 1]
```diff
@@ -176,7 +176,7 @@ Public MustInherit Class BasicTestBase
 
         Dim assemblyName As String = Nothing
         Dim sourceTrees = ParseSourceXml(source, parseOptions, assemblyName)
-        Dim compilation = CreateEmptyCompilation(sourceTrees.ToArray(), allReferences, options, assemblyName:=assemblyName)
+                        Dim compilation = CreateEmptyCompilation(sourceTrees.ToArray(), allReferences, options, parseOptions, assemblyName:=assemblyName)
```
Comment (CyrusNajmabadi) [inline src/Compilers/Test/Utilities/VisualBasic/BasicTestBase.vb:179]: please don't change formatting unnecessarily

Comment (AdamSpeight2008) [reply src/Compilers/Test/Utilities/VisualBasic/BasicTestBase.vb:179]: It the online editor cause the formatting change, I go an fix it.

File: src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb [hunk 2]
```diff
@@ -43,6 +43,11 @@ Friend Module CompilationUtils
         If options Is Nothing Then
             options = TestOptions.ReleaseDll
         End If
+        
+        ' update the parseOption (which ensures the language versions a consistant
```
Comment (jaredpar) [inline src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb:47]:
```suggestion
```

Don't think the comment is necessary here.

Comment (333fred) [reply src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb:47]: The trailing whitespace is also what is failing the formatter.

Comment (AdamSpeight2008) [reply src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb:47]: @jaredpar I think the comment is important, as the potential inconsistency isn't documented.
@333fred  It the GitHub online editor, that cause the issues around whitespace.

Comment (jaredpar) [reply src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb:47]: This is a pretty common pattern in our code and virtually none of the other occurrences warrant a comment.

**Pull request closed without merging**